doc/lua_api.txt

```diff
@@ -1398,6 +1398,16 @@ For helper functions see "Vector helpers".
 * `{type="node", under=pos, above=pos}`
 * `{type="object", ref=ObjectRef}`
 
+Exact pointing location (currently only `Raycast` supports these fields):
+* `pointed_thing.intersection_point`: The point on the object's selection box
+  which is pointed at. May be in the selection box if the pointer is in
+  the box too.
+* `pointed_thing.box_id`: The ID of the pointed selection box (counting starts
+  from 0).
+* `pointed_thing.intersection_normal`: Unit vector, points outwards of the
+  selected selection box. This specifies which face is pointed at.
+  Null vector, if the pointer is inside the selection box.
```

Comment:
What is a null vector?  Probably `{x=0,y=0,z=0}`, but this terminology isn't used anywhere else